src/mokka/src/main/java/pl/hycom/mokka/service/openapi/DefaultOpenApiParsingService.java

```diff
@@ -0,0 +1,49 @@
+package pl.hycom.mokka.service.openapi;
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.parser.OpenAPIV3Parser;
+import io.swagger.v3.parser.core.models.SwaggerParseResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.stereotype.Service;
+import pl.hycom.mokka.service.openapi.pojo.StubMappingListWrapper;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Kamil Adamiec (kamil.adamiec@hycom.pl)
+ */
+@Service
+public class DefaultOpenApiParsingService implements OpenApiParsingService {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DefaultOpenApiParsingService.class);
+
+    @Autowired
+    @Qualifier("mvcConversionService")
```

Comment:
Constructor-based injection is preferred: remove Autowired, add `final` and add Lombok's `@RequiredArgsConstructor` on class level. `ConversionService` is registered via: ConversionConfiguration